CONFIG.md

```diff
@@ -0,0 +1,232 @@
+Configuration
+---------------------------------
+
+This document provides guidance to configuring an inference pipeline with ltu-ili and includes links and references to the various functionality options. There are three stages to the inference pipeline:
+- [**Data Loading**](#data-loading): Loading various structured data into memory
+- [**Training**](#training): Training neural networks from the loaded data and saving them to file.
+- [**Validation**](#validation): Loading neural networks from file, sampling posteriors on the test set, and computing metrics.
+
+In general, these steps are designed to be independent. For example, if you wanted to try out a different training model, all you would need to do is change the training configuration, and the data loading and validation stages should integrate (nearly) seamlessly with the new model. Each pipeline stage specified below can be instantiated from `json`-like configuration files or from iPython initialization as in [tutorial.ipynb](notebooks/tutorial.ipynb).
+
+## Data Loading
+We have three primary objects for dataloading:
```

Comment:
There are four bullet points but you say there are three primary objects. Is one of these deemed to not be primary or should this just be four? If the former, please clarify as it's a little confusing

Reply:
:) This was before we merged the jupyter interface. Changed to four.